Client/Frontend/Browser/EnhancedTrackingProtection/EnhancedTrackingProtectionVM.swift

```diff
@@ -31,13 +31,12 @@ class EnhancedTrackingProtectionMenuVM {
     }
 
     var connectionStatusImage: UIImage {
-        let insecureImageString = "lock_blocked"
-        let image = connectionSecure ? UIImage(named: "secureLock")! : UIImage(imageLiteralResourceName: insecureImageString)
-        return image
+        let connectionStatus: WebsiteConnectionTypeStatus = connectionSecure ? .secure : .unsecure
+        return ConnectionStatusImage.getForStatus(status: connectionStatus)!
     }
 
     var connectionSecure: Bool {
-        return tab.webView?.hasOnlySecureContent ?? false
+        return tab.webView?.url?.isHTTPS == true
```

Comment:
👏  Very nice!

I wonder if we'd even go for a three-stage icon:

Insecure, Mixed and `hasOnlySecureContent`...

Actually, that said I'm not even sure if the WebView loads insecure content if the root site is secure.

Reply:
Yeah I wasn't 💯 sure about it, but was I 💯 sure of is the fact that `hasOnlySecureContent` isn't actually the best check as some underlying content (like images or resources) may not be under `https` or secure connetions